gnd/src/main/java/com/google/android/gnd/ui/projectselector/ProjectSelectorViewModel.java

```diff
@@ -15,48 +15,89 @@
  */
 package com.google.android.gnd.ui.projectselector;
 
+import android.util.Log;
+
 import androidx.lifecycle.LiveData;
 import androidx.lifecycle.MutableLiveData;
+
 import com.google.android.gnd.repository.DataRepository;
 import com.google.android.gnd.repository.Resource;
+import com.google.android.gnd.rx.Result;
 import com.google.android.gnd.system.AuthenticationManager;
 import com.google.android.gnd.ui.common.AbstractViewModel;
 import com.google.android.gnd.vo.Project;
-import io.reactivex.Completable;
+
+import io.reactivex.Observable;
+import io.reactivex.subjects.PublishSubject;
+
 import java.util.Collections;
 import java.util.List;
+
 import javax.inject.Inject;
 
 public class ProjectSelectorViewModel extends AbstractViewModel {
   private static final String TAG = ProjectSelectorViewModel.class.getSimpleName();
 
   private final DataRepository dataRepository;
   private final MutableLiveData<Resource<List<Project>>> projectSummaries;
-  private final AuthenticationManager authManager;
+  private final PublishSubject<Integer> projectSelections;
+  public final MutableLiveData<Project> activeProject;
+  public final MutableLiveData<Throwable> activateProjectErrors;
+  public final Observable<Result<Project>> activeProjectStream;
 
   @Inject
   ProjectSelectorViewModel(DataRepository dataRepository, AuthenticationManager authManager) {
     this.dataRepository = dataRepository;
     this.projectSummaries = new MutableLiveData<>();
-    this.authManager = authManager;
-  }
+    this.activeProject = new MutableLiveData<>();
+    this.activateProjectErrors = new MutableLiveData<>();
+    this.projectSelections = PublishSubject.create();
+
+    AuthenticationManager.User user =
+        authManager.getUser().blockingFirst(AuthenticationManager.User.ANONYMOUS);
```

Comment:
Are we okay with using [`blockingFirst`](http://reactivex.io/RxJava/javadoc/io/reactivex/Observable.html#blockingFirst-T-)? It's the most concise way, I think, of doing what we want here (getting and keeping a reference only to the first user returned by the user stream, that way the user referenced throughout the interaction is the same).

There are alternatives, but they involve a bit more manipulation since `getUser` returns an Observable.

Reply:
In the future would like to expose a synchronous `currentUser()` method and create the backing Behavior with default anon user, but for the purposes of this fix it looks like the right thing.

Reply:
Sounds good! #86!